### PR TITLE
added Header and Footer

### PR DESCRIPTION
**Pull request by vineethvanga18**

### CODE REVIEW

File: frontend/src/App.vue [hunk 1]
```diff
@@ -1,8 +1,24 @@
 <template lang="pug">
-  div(id="app")
-    router-view
+  v-app(style="background-color: white")
+    PageHeader
+    v-content
+      v-container(fluid :fill-height="$route.meta.fillHeight")
```
Comment (Tanuj22) [inline frontend/src/App.vue:5]: No use of `:fill-height="$route.meta.fillHeight"`

File: frontend/src/App.vue [hunk 2]
```diff
@@ -1,8 +1,24 @@
...
+export default {
+  name: "App",
+  components: {
+    PageHeader,
+    PageFooter
+  }
+};
+</script>
 <style>
 #app {
```
Comment (Tanuj22) [inline frontend/src/App.vue:23]: Is this of any use?

File: frontend/src/components/Footer.vue [hunk 3]
```diff
@@ -0,0 +1,17 @@
+<template lang="pug">
+  v-footer(color="#000080" padless)
```
Comment (Tanuj22) [inline frontend/src/components/Footer.vue:2]: Define any new color in https://github.com/devlup-labs/portfolio/blob/master/frontend/src/plugins/vuetify.js

File: frontend/src/components/Footer.vue [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+    v-row(justify="center" no-gutters)
+      v-btn(v-for="link in links" :key="link" color="white" text rounded class="py-2") {{ link }}
```
Comment (Tanuj22) [inline frontend/src/components/Footer.vue:4]: Use pug syntax for `class`

File: frontend/src/components/Footer.vue [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+    v-col(style="background-color:#000099" class="py-4 text-center white--text" cols="12") {{ new Date().getFullYear() }} —
```
Comment (Tanuj22) [inline frontend/src/components/Footer.vue:5]: Same as above two

File: frontend/src/components/Footer.vue [hunk 6]
```diff
@@ -0,0 +1,17 @@
+<template lang="pug">
+  v-footer(color="#000080" padless)
+    v-row(justify="center" no-gutters)
+      v-btn(v-for="link in links" :key="link" color="white" text rounded class="py-2") {{ link }}
+    v-col(style="background-color:#000099" class="py-4 text-center white--text" cols="12") {{ new Date().getFullYear() }} — 
+        strong Devlup Labs
```
Comment (Tanuj22) [inline frontend/src/components/Footer.vue:6]: Use `font-weight-bold` class of vuetify in place of strong

File: frontend/src/components/Header.vue [hunk 7]
```diff
@@ -0,0 +1,21 @@
+<template lang="pug">
+    v-app-bar(app clipped-left color="#000080")
+        v-avatar(size="60px")
+            img(src="../assets/logo.jpeg")
+        v-toolbar-title.white--text
+            span.title.ml-3.mr-5 Devlup Labs&nbsp;
+        v-spacer
+        v-btn(color="white" text rounded class="my-2") Home
```
Comment (Tanuj22) [inline frontend/src/components/Header.vue:8]: Use pug for `class`

File: frontend/src/components/Header.vue [hunk 8]
```diff
@@ -0,0 +1,21 @@
+<template lang="pug">
+    v-app-bar(app clipped-left color="#000080")
```
Comment (Tanuj22) [inline frontend/src/components/Header.vue:2]: Define any new color in https://github.com/devlup-labs/portfolio/blob/master/frontend/src/plugins/vuetify.js

**Review by Tanuj22 — CHANGES_REQUESTED** [review]
The page is not responsive. Hide the tabs in small screen and add a hamburger

File: frontend/src/App.vue (hunk 2 above)
Comment (vineethvanga18) [reply frontend/src/App.vue:23]: Header and footer are to be shown in every page so making it global

**Pull request merged**